packages/stepper/src/Stepper.story.tsx

```diff
@@ -13,21 +13,24 @@ storiesOf('Stepper', module)
     <div style={{ width: 1000 }}>
       <Stepper
         currentStep={number('Step', 0, { min: 0, max: 6 })}
+        completedStepsShown={number('Completed steps shown', 2, { min: 1 })}
         maxDisplayedSteps={5}
       >
-        <Step>Overview</Step>
-        <Step>Configuration</Step>
-        <Step>Update</Step>
-        <Step>Install</Step>
-        <Step>Billing</Step>
-        <Step>Address</Step>
-        <Step>Confirmation</Step>
+        <div>Overview</div>
+        <div>Configuration</div>
+        <div>Update</div>
+        <div>Install</div>
+        <div>Billing</div>
+        <div>Address</div>
+        <div>Confirmation</div>
       </Stepper>
     </div>
   ))
   .add('Many', () => {
-    let currentStep = number('Step', 0, { min: 0 });
-    const numSteps = number('Number of steps', 7, { min: 1 });
+    let currentStep = number('Step', 1, { min: 0 });
+    const numSteps = number('Number of steps', 10, { min: 1 });
+    const maxDisplayedSteps = number('Max displayed', 5, { min: 1 });
+    const completedStepsShown = number('Completed steps shown', 3, { min: 1 });
```

Comment:
This is a new optional prop that changes where the current step is rendered.

Below is what the previous rendering logic would default to. In the new implementation, this is equivalent to `completedStepsShown = 1`, which is the default value.
![image](https://user-images.githubusercontent.com/99368875/157321284-bf23d1de-abd0-437a-bf09-a48032f18d0c.png)

But since the stepper has 5 visible steps, it's probably nicer to render the current step in the center like below. This is `completedStepsShown = 2`
![image](https://user-images.githubusercontent.com/99368875/157321319-1c83d97f-5821-4c05-bd71-4c8a886dc9af.png)